### ISSUE TITLE
yodaConditions breaks expressions with ternary operator

### ISSUE THREAD

**Issue by ChristopherRogers** [nicklockwood/SwiftFormat#392]
Before:
```swift
let y = 9
let z = 0 < y ? 3 : 4
```

After:
```swift
let y = 9
let z = y ? 3 : 4 > 0
```

**Comment by nicklockwood** [comment]
@christopherrogers thanks, I've pushed a fix to the develop branch.

**Comment by nicklockwood** [comment]
@ChristopherRogers fixed in 0.40.3.
